### DISCUSSION TITLE
📖 Type of Reentrancy Attack

### DISCUSSION THREAD

**Discussion by rokk1t** [pcaversaccio/reentrancy-attacks#24] [Q&A] [answered]
If possible, could you tell me what type of reentrancy attack each case belongs to?

**Comment by pcaversaccio** [comment, marked as answer]
> If possible, could you tell me what type of reentrancy attack each case belongs to?

This is left to the reader / learner as an exercise :).

**Comment by Ray1010101** [comment]
A reentrancy attack that's what I want to run can you give me a guide?

**Reply by pcaversaccio** [reply]
Lol no

**Reply by Ray1010101** [reply]
Why no?
